onnxscript/test/function_libs/torch_aten/ops_correctness_test.py

```diff
@@ -189,18 +197,22 @@ def wrapped(fn):
     "reciprocal": core_ops.aten_reciprocal,
     "remainder": core_ops.aten_remainder,
     "repeat": core_ops.aten_repeat,
+    "reshape": core_ops.aten_reshape,
     "round": core_ops.aten_round,
     "rsqrt": core_ops.aten_rsqrt,
     "sigmoid": core_ops.aten_sigmoid,
     "sign": core_ops.aten_sign,
     "sin": core_ops.aten_sin,
     "sinh": core_ops.aten_sinh,
+    "slice": core_ops.aten_slice,
     "sqrt": core_ops.aten_sqrt,
     "sub": core_ops.aten_sub,
+    # "sum": core_ops.aten_sum, #TODO: kwargs={dim, keepdims}, dim is invalid
```

Comment:
Both dim and keepdims exisit in the current symbolic function in pytorch. We should be able to support both.